### DISCUSSION TITLE
Why can't it reach the maximum speed of the network card?

### DISCUSSION THREAD

**Discussion by taomujian** [projectdiscovery/naabu#754] [Q&A]
I use naabu to scan the port and set the speed to be very high. The network card is a Gigabit network card, and the real-time traffic reaches a maximum of 20M/s, but I use masscan to reach a maximum traffic of 100M/s. Why?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
